packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -169,6 +170,27 @@ If a misconfigured constraint is added, `synth` will fail with an error message.
 
 Read more at [Service Catalog Constraints](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints.html).
 
+### Event notification constraint
+
+An event notification constraint allows users to subscribe a list of AWS `SNS` topics to the stack events of the product.
```

Comment:
🙃 That's just how service catalog is designed.  I am fine with renaming obviously, but want to maintain some consistency to the console and cli experience.  In the `README` I tried to convey this with defining constraints as operations on portfolio-product association.
Stack events are just standard stack lifecycle events - create, update , delete for the product stack

Reply:
> Stack events are just standard stack lifecycle events - create, update , delete for the product stack.

So how does this work for non-CloudFormation Products?

Reply:
there are no non-Cloudformation products (yet).  Every time a user provisions a product it creates the stack that these topics are then subscribed to.

Reply:
> there are no non-Cloudformation products (yet).

So, in the [ReadMe of this module](https://github.com/aws/aws-cdk/blob/ebfc0e092addbcf21562ce63328457285f7b2d77/packages/@aws-cdk/aws-servicecatalog/README.md#product), there is this sentence:

> The CDK currently only supports adding products of type Cloudformation product. 

This implies that, even if none exist today, there will be different types of Products, right? Given that, is it a good idea to add these Product-specific constraints to Portfolio?

Reply:
If/when a new product type comes out, we will be adding constraints again at the association level, and service catalog will try to support as many of the current constraints as possible for new product types to avoid a regression in functionality.  Even for bidirectional capabilities, we will still have a portfolio as a part of the argument since that is how SC fundamentally still works, and the mental model is to generally operate from the portfolio side.  We can strongly type inputs on portfolio methods if there is incompatibility between product types.

Reply:
OK. If you're fine having Portfolio have the sum of all possible constraints of all types of Products in its API, then so be it. This is not the choice I would have made, but I won't hold the PR on it.

Reply:
I think for foreseeable future this will be fine and unlikely to change.  Also since still experimental it's a 2 way door right?    Our goal is to get some real user feedback to guide refinements.

Reply:
Sure.